lib/modules/hosts/bime.js

```diff
@@ -0,0 +1,13 @@
+export default {
+	moduleID: 'bime',
+	name: 'Bime Analytics Dashboards',
+	domains: ['bime.io'],
+	logo: '//a.bime.io/assets/favicons/favicon.ico',
+	detect: ({ href }) => (/https?:\/\/([^.]+)\.bime\.io(?:\/([a-z0-9_-]+))+/i).exec(href),
+	handleLink: (href, [, user, dashboardId]) => ({
+		type: 'IFRAME',
```

Comment:
Looks great, just one last thing:

Can you add `expandoClass: 'selftext',` here, so it has a text icon?